someengineering/resoto/templates/resotocore/deployment.yaml

```diff
@@ -72,6 +72,10 @@ spec:
           {{- if .Values.resotocore.extraEnv }}
           {{- toYaml .Values.resotocore.extraEnv | nindent 12 }}
           {{- end }}
+          - name: HELM_VERSION
```

Comment:
What is the idea behind this change? Why do we want to embed the chart_version when the chart is already versioned 🤔

Reply:
So that the core knows whether it's running in a Helm installed environment or if someone transformed the docker-compose and loaded that into their k8s cluster.

Reply:
So how does it affect the core behaviour if I use the kubernetes-manifest via the image or use the helm-chart? Are we planning to have some additional scraping on the basis of these env variables. Or if it is needed to check the installation method for debugging we can always get `helm list` to know if it was done via helm-chart.

Reply:
`helm list` can't be executed by the core though can it?

Reply:
I'm not sure what you mean by "additional scraping". The core uses the environment information to know which kind of environments errors happen in for instance. So for instance, if we see an increase in certain exceptions it's useful to know if it happens in all environments or only certain ones.